ports/servoshell/egl/app_state.rs

```diff
@@ -337,6 +337,19 @@ impl RunningAppState {
         self.inner_mut().webviews.insert(webview.id(), webview);
     }
 
+    pub(crate) fn activate_webview(&self, id: u32) {
```

Comment:
```suggestion
    pub(crate) fn focus_webview(&self, id: WebViewId) {
```

Reply:
I changed the name of the function but kept the id for now.

Reply:
I think it makes sense to pass `WebViewId` and to store it everywhere in order to make the type of the id clear.

Reply:
Ok I changed it now and I think this makes more sense. One snag that I experienced was that the active webview changes quite slowly in RunningAppState. For this reason we now need access to the newest_webview.